libs/dh/metering-point/feature-move-in/src/components/dh-address-details.component.ts

```diff
@@ -0,0 +1,205 @@
+              [label]="t('floor')"
+              data-testid="floor"
+            />
+            <watt-text-field
+              [formControl]="groupControls.room"
+              [label]="t('door')"
+              data-testid="room"
+            />
+          </vater-flex>
+
+          <vater-flex direction="row" gap="m" justify="space-between">
+            <watt-text-field
+              [formControl]="groupControls.postCode"
+              [label]="t('postalCode')"
+              class="flex-grow-1"
+              data-testid="post-code"
+            />
+            <watt-text-field
+              [formControl]="groupControls.cityName"
+              [label]="t('city')"
+              class="flex-grow-2"
+              data-testid="city-name"
+            />
+
+            <watt-text-field
+              [formControl]="groupControls.citySubDivisionName"
+              [label]="t('citySubdivisionName')"
+              class="flex-grow-2"
+              data-testid="city-subdivision-name"
+            />
+          </vater-flex>
+
+          <vater-flex direction="row" gap="m" justify="space-between">
+            <watt-text-field
+              [formControl]="groupControls.streetCode"
+              [label]="t('streetCode')"
+              data-testid="street-code"
+            />
+            <watt-text-field
+              [formControl]="groupControls.municipalityCode"
+              [label]="t('municipalityCode')"
+              maxLength="3"
+              data-testid="municipality-code"
+            >
+              <watt-field-error>
+                @if (groupControls.municipalityCode.hasError('containsLetters')) {
+                  {{ t('municipalityCodeError.containsLetters') }}
+                } @else if (groupControls.municipalityCode.hasError('startsWithZero')) {
+                  {{ t('municipalityCodeError.startsWithZero') }}
+                } @else if (groupControls.municipalityCode.hasError('invalidMunicipalityCodeLength')) {
+                  {{ t('municipalityCodeError.invalidMunicipalityCodeLength') }}
+                }
+              </watt-field-error>
+            </watt-text-field>
+          </vater-flex>
+
+          <vater-flex direction="row" gap="m" justify="space-between">
+            <watt-text-field
+              [formControl]="groupControls.postalDistrict"
+              [label]="t('postalDistrict')"
+              data-testid="postal-district"
+            />
+            <watt-text-field
+              [formControl]="groupControls.postBox"
+              [label]="t('postBox')"
+              data-testid="post-box"
+            />
+          </vater-flex>
+
+          <watt-text-field
+            [formControl]="groupControls.darReference"
+            [label]="t('darReference')"
+            data-testid="dar-reference"
+          />
+
+          <watt-checkbox
+            [formControl]="formGroup.controls.nameAddressProtection"
+            class="checkbox-margin-bottom"
```

Comment:
suggestion: let's use the `watt-` util classes to add bottom margin. Remember to remove the custom class from `styles`.


```suggestion
            class="watt-space-stack-l"
```